website/docs: add a single page about our user interface, document Consent stage (#20533)
goauthentik/authentik: website/docs/users-sources/user/user-interface.mdx

---

Administrators for an authentik instance work primarily in the Admin interface; this is where administrators create new users and groups, add applications, and manage system settings.

End-users, who are accessing their applications via authentik for their daily tasks, typically only access the User interface. (There are exceptions; some end-users have [permissins to also access the Admin interface](https://docs.goauthentik.io/users-sources/access-control/manage_permissions/#assign-can-view-admin-interface-permissions), and some end-users never go to the User interface, but rather [log directly into their application](https://docs.goauthentik.io/brands/#external-user-settings) and login using authentik behind the scenes.)

:::info
This document covers the basic tasks that end-users accomplish in the User interface. All of our technical documentation is available to all users, just note that the vast majority of it is for the Admin interface because the User interface is for simple end-user tasks.
:::

## User details

words here

## Sessions

words here... one can delete sessions even the current one which logs them out.. or a session on a remote device

## Consent

> “related to the stage. operating modes: you have an auth flow, about to loginto grafana, with OAuth the app can request account info.. profile and email address … if you have a consent stage, it pops up and says Hey, the app is asking for the data, do you want to share this info…? If Yes, click Continue. In this stage there are three modes: 1) always require (the user will see message and consent by cliing continue 2) Permanent: saves fact that you clicked on Continue, creates a Consent object with alink to the user and to the app,, and stores which data (but not value), Expiring: means you have to click Continue anytime after the expiry date stage set in stage as Duration.

Reviewer: ```suggestion
“related to the stage. operating modes: you have an auth flow, about to log in to grafana, with OAuth the app can request account info.. profile and email address … if you have a consent stage, it pops up and says Hey, the app is asking for the data, do you want to share this info…? If Yes, click Continue. In this stage there are three modes: 1) always require (the user will see message and consent by clicking continue 2) Permanent: saves fact that you clicked on Continue, creates a Consent object with a link to the user and to the app, and stores which data (but not value), Expiring: means you have to click Continue anytime after the expiry date stage set in stage as Duration.
```

To view your user settings, click the gear icon in the upper right. The following settings are displayed:

### User details

This section of the User interface displays (and allows you to edit if you have the requisite permissions) the following fields:

- **Username**: the username is the unique identifier associated with the user, and is required for logging in. This value can only be edited by the user if the [global System settings](../../sys-mgmt/settings.md#allow-users-to-change-username) are configured to allow all users to change their username, or if the attribute [`can-change-username`](../../users-sources/user/user_ref.mdx#attributes) has been added to the **Attributes** field for a specific user (overriding the global System setting).
- **Name**: a display name, or nickname, for the user. Similar to the username, this value can be set globally in [System settings](../../sys-mgmt/settings.md#allow-users-to-change-name), or per user with the attribute [`goauthentik.io/user/can-change-name`](../../users-sources/user/user_ref.mdx#attributes).

> - **Email**: the email address for the user. This value also can be set globally in [System settings](../../sys-mgmt/settings.md#allow-users-to-change-email), or [per user](../../users-sources/user/user_ref.mdx#attributes).

Author: rewrote already

- **Locale**: override any global locale settings and either choose a specific language or select **Auto-Detect**, which relies on the user's browser local settings.
- **Change your password**: if a user has the [permission to update their password](../user/user_basic_operations.md#reset-a-password), they can do so here.

### Sessions

This tab shows all active sessions for the user. Here you can delete sessions, including the current one (which would result in an automatic log out) or a session on a remote device.

### Consent

> You can view applications that are configured to use the Consent stage, and which you consented to allow authentik to share your profile user data with the application.

Reviewer 2: ```suggestion
You can view applications to which you gave consent to allow authentik to share your profile user data with the application.
```

> When an administrator adds this stage to an authorization flow, the user logging in is presented with a pop-up confirmation page asking if they agree to allow the application (the relying party or RP) to directly request their account data (typically profile and email address) from the source. The user clicks **Continue** to give consent.

Reviewer 2: ```suggestion
When an administrator adds this stage to an authorization flow, the user logging in is presented with a pop-up confirmation page asking if they agree to allow the application to directly request their account data (typically profile and email address) from the source. The user clicks **Continue** to give consent.
```

For more information refer to our documentation on the [Consent stage](../../add-secure-apps/flows-stages/stages/consent/index.md).

### MFA Devices

This is where a users can add and configure a new MFA device for accessing authentik. The three supported options for MFA are:

- **Static tokens**: authentik generates 6 single-use tokens.
- **TOTP device**: using your preferred authenticator, scan the QR code, enter the code from the authenticator into the authentik prompt, and then click **Continue**. For authenticators that do not support QR scanning, you can copy the secret and paste it into you authenticator.
- **WebAuthn device**: this option uses the [WebAuthn/FIDO2/Passkeys Authenticator setup stage](../../add-secure-apps/flows-stages/stages/authenticator_webauthn/index.mdx) to allow the user to create a passkey for the device.

> 

Reviewer 2: - missing SMS
- missing Duo
- missing Email

:::info LDAP providers and MFA

> Because LDAP does not natively support OTP, authentik supports [code-based MFA](../../add-secure-apps/providers/ldap/index.md#code-based-mfa-support) for situations where the protocol is LDAP and they are required to use MFA. If enabled, the user can enter the authenticator's code as part of the bind/authentication password, separated by a semicolon. For example, for the password `example-password` and the MFA code `123456`, the input in the password field must be `example-password;123456`.

Reviewer 2: ```suggestion
Because LDAP does not natively support OTP, authentik supports [appending the OTP code to the password](../../add-secure-apps/providers/ldap/index.md#code-based-mfa-support) for situations where the protocol is LDAP and they are required to use MFA. If enabled, the user can enter the authenticator's code as part of the bind/authentication password, separated by a semicolon. For example, for the password `example-password` and the MFA code `123456`, the input in the password field must be `example-password;123456`.
```
